### DISCUSSION TITLE
Deployed help system content generation from github wiki-based User Guide

### DISCUSSION THREAD

**Discussion by tparchambault** [ctc-oss/fapolicy-analyzer#764] [Developers]
The AAC developers (primarily SD and JW) have introduced automated population of the deployed application's Help content. There is currently a development effort to strengthen the sync between the Users Guide content in the wiki and fapolicy-analyzer application's release features and implementation details e.g. the images and/or user workflow, configuration settings, etc as presented in the Users Guide are accurate with respect to its associated application release.

**Comment by tparchambault** [comment]
Q. @jw3 How (and in which repo) do we want to commit the Users Guide gfx? In the main application repo after help content regeneration (similar to pot file generation, on an as-needed basis) or in the current wiki location?

**Reply by jw3** [reply]
Images go in the wiki repo.  Add a directory named images.  Then hrefs to these in the markdown will use relative paths.

**Comment by jw3** [comment]
The overall task here should be roughly

1. Locate where the hrefs are parsed
2. Add an if statement that checks if the href starts with "http"
   - if it does, do what is done currently, that is download to output dir
   - if it does not then assume the image is relative and copy from relative path to output dir


Shouldn't take much more than that.  Roughly.

**Reply by jw3** [reply]
The issue for this is #756
